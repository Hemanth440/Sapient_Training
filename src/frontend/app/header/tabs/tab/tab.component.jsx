import React from 'react';
import {NavLink} from "react-router-dom";

export const Tab = ({tab}) =>
    <li className="tabs-list__item">
        <NavLink to={tab.url} exact={tab.isRootUrl} activeClassName="active">
            {tab.name}
        </NavLink>
    </li>
