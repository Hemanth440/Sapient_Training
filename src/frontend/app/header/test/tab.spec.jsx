import React from 'react';
import {shallow} from "enzyme";
import {Tab} from "../tabs/tab/tab.component";

describe('<Tab/>', () => {
    it('should contain NavLink', () => {
        const tab = {
            isRootUrl: false,
            url: '/'
        };
        const wrapper = shallow(<Tab tab={tab}/>);
        expect(wrapper.find('NavLink')).toMatchSnapshot();
    });
});
