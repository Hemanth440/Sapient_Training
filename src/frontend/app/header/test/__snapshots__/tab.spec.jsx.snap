// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Tab/> should contain NavLink 1`] = `
<NavLink
  activeClassName="active"
  ariaCurrent="true"
  exact={false}
  to="/"
/>
`;
